src/ncp/ncp_uart.cpp

```diff
@@ -270,12 +270,101 @@ void NcpUart::HandleError(ThreadError aError, uint8_t *aBuf, uint16_t aBufLength
 #ifdef __cplusplus
 extern "C" {
 #endif
-void otCliLog(otLogLevel aLogLevel, otLogRegion aLogRegion, const char *aFormat, va_list aAp)
+
+#define LOG_APPEND(CONST_STR) \
+    strcpy(logString + offset, CONST_STR); \
+    offset += sizeof(CONST_STR) - 1; \
+
+void otCliLog(otLogLevel aLogLevel, otLogRegion aLogRegion, const char *aFormat, va_list args)
 {
-    (void)aLogLevel;
-    (void)aLogRegion;
-    (void)aFormat;
-    (void)aAp;
+    char logString[128];
+    unsigned int offset = 0;
+
+    switch (aLogLevel)
+    {
+    case kLogLevelNone:
```

Comment:
I was just keeping with the format the CLI logging was doing. I am fine with changing though.